content/blog/september-update.md

```diff
@@ -0,0 +1,37 @@
+---
+title: September Update
+date: '2020-09-10T10:20:39-04:00'
+author: DJ
+last_edited: '2020-09-10T15:10:14.859Z'
+---
+👋 Hey all you cool cats and kittens! It's been a while since our last update, so we wanted to let you know what we've been up to.
+
+## Documentation Cycle Recap
+
+Tina is a toolkit for developers, so documentation is essential. If a feature is not well-documented, it might as well not exist. In light of this, we've spent the past several weeks improving the usability of the documentation on [tinacms.org](). We've reorganized the docs and written some new ones, improved the functionality of our search, and are now importing the package docs [directly from GitHub]() to ensure they are always in sync.
```

Comment:
```suggestion
Tina is a toolkit for developers, so documentation is essential. If a feature is not well-documented, it might as well not exist. In light of this, we've spent the past several weeks improving the usability of the documentation on [tinacms.org](https://tinacms.org). We've reorganized the docs and written some new ones, improved the functionality of our search, and are now importing the package docs [directly from GitHub](/packages/) to ensure they are always in sync.
```